source/documentation/deploying_services/sqs.md

```diff
@@ -0,0 +1,350 @@
+# Amazon SQS
+
+Amazon Simple Queue Service (SQS) offers a durable hosted queue that lets you
+build decoupled distributed software components.
+
+To provide any feedback on this feature, contact the GOV.UK PaaS team at
```

Comment:
Do we still need this line about feedback, or is the one about testing the service supposed to replace it?

Reply:
I don't think we need both